modules/account-lib/src/coin/algo/keyRegistrationBuilder.ts

```diff
@@ -177,6 +196,7 @@ export class KeyRegistrationBuilder extends TransactionBuilder {
         this.voteFirst(algoTxn.voteFirst);
         this.voteLast(algoTxn.voteLast);
         this.voteKeyDilution(algoTxn.voteKeyDilution);
+        algoTxn.stateProofKey ? this.stateProofKey(algoTxn.stateProofKey.toString('base64')) : null;
```

Comment:
This can be written as:
```suggestion
        algoTxn.stateProofKey && this.stateProofKey(algoTxn.stateProofKey.toString('base64'));
```
But I don't think it is a TypeScript or Javascript idiom.

This is:
```suggestion
        if (algoTxn.stateProofKey) {
          this.stateProofKey(algoTxn.stateProofKey.toString('base64'));
        }
```